library/src/kinectDriverOpenNI.cpp

```diff
@@ -89,29 +89,29 @@ bool KinectDriverOpenNI::initialize(Property &opt)
     this->seatedMode=false;
     this->img_width=opt.check("img_width",Value(320)).asInt();
     this->img_height=opt.check("img_height",Value(240)).asInt();
+    this->depth_width=opt.check("depth_width",Value(320)).asInt();
```

Comment:
Hi, the options were already documented in `modules/kinectServer/src/main.cpp`. To be consistent, I also added them in ` library/include/kinectWrapper/kinectDriver.h` (and fixed `image_width` -> `img_width`). Respective commit: https://github.com/Tobias-Fischer/kinect-wrapper/commit/c2dba62f917b56a2d19c1a58344c92c137170d2c